doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -1687,6 +1687,10 @@ def build_inconsistency_annotations(inconsistencies: Iterable[AssemblyIssue]) ->
         exclude_assembly_issues = [AssemblyIssueCode.OUTDATED_RPMS_IN_STREAM_BUILD]
 
         msgs = sorted([i.msg for i in inconsistencies if i.code not in exclude_assembly_issues])
+
+        if not msgs:
+            return
```

Comment:
I think this function should return a dict, which is expected from the caller. Returning None could cause trouble. See line #1684